outbox-kafka-spring/src/test/java/one/tomorrow/transactionaloutbox/service/OutboxProcessorIntegrationTest.java

```diff
@@ -206,7 +201,7 @@ public void should_ContinueProcessingAfterDbConnectionFailureInLockAcquisition()
         CountDownLatch cdl = new CountDownLatch(1);
 
         OutboxLockRepository failingLockRepository = (OutboxLockRepository) beanFactory.applyBeanPostProcessorsAfterInitialization(
-                new OutboxLockRepository(sessionFactory) {
+                new OutboxLockRepository() {
```

Comment:
I think this now needs to get an `entityManager` (which was a `sessionFactory` before).

Reply:
I think this is because of the way the `OutboxProcessor` instantiates its fields. It works with the `DefaultOutboxEntityManager` still existing, that's why I undid my last commit that removed it.

I see two options:

1. Keep the `DefaultOutboxEntityManager` for now.
2. Remove it, but then also change the way the `OutboxProcessor` instantiates its fields.

I'd lean towards option 1, leaving option 2 as a mid-term goal we want to achieve, but not necessarily in this PR. Would that work for you or would option 2 be necessary for you to approve this PR?